--- origsrc/libcuefile/CMakeLists.txt	2010-01-19 19:30:20.000000000 -0600
+++ src/libcuefile/CMakeLists.txt	2010-01-19 19:35:51.775779200 -0600
@@ -5,3 +5,4 @@ project(libcuefile C)
 set(CMAKE_VERBOSE_MAKEFILE false)
 
 add_subdirectory(src)
+add_subdirectory(include)
--- origsrc/libcuefile/include/CMakeLists.txt	1969-12-31 18:00:00.000000000 -0600
+++ src/libcuefile/include/CMakeLists.txt	2010-01-19 19:35:26.936358500 -0600
@@ -0,0 +1 @@
+INSTALL(DIRECTORY cuetools DESTINATION include)
--- origsrc/libcuefile/src/CMakeLists.txt	2010-01-19 19:30:20.000000000 -0600
+++ src/libcuefile/src/CMakeLists.txt	2010-01-19 19:31:16.394028300 -0600
@@ -7,4 +7,4 @@ set_target_properties(cuefile-static PRO
 add_library(cuefile-shared SHARED cd cdtext cue_parse cue_print cue_scan cuefile time toc toc_parse toc_print toc_scan)
 set_target_properties(cuefile-shared PROPERTIES OUTPUT_NAME cuefile CLEAN_DIRECT_OUTPUT 1 VERSION 0.0.0 SOVERSION 0)
 
-install(TARGETS cuefile-static cuefile-shared LIBRARY DESTINATION "lib${LIB_SUFFIX}" ARCHIVE DESTINATION "lib${LIB_SUFFIX}")
+install(TARGETS cuefile-static cuefile-shared LIBRARY DESTINATION "lib${LIB_SUFFIX}" ARCHIVE DESTINATION "lib${LIB_SUFFIX}" RUNTIME DESTINATION "bin")
